diff --git a/src/Tools/GetVersions.php b/src/Tools/GetVersions.php
index 429f011..789028b 100644
--- a/src/Tools/GetVersions.php
+++ b/src/Tools/GetVersions.php
@@ -41,9 +41,10 @@ class GetVersions
         }
 
         $installedTools = json_decode((string) file_get_contents($installedJson));
-        if (!is_array($installedTools)) {
+        if (!isset($installedTools->packages)) {
             return [];
         }
+        $installedTools = $installedTools->packages;
 
         $tools = array();
         foreach ($installedTools as $tool) {
